uint8 id
uint8 cmd_mode
float32 torque